cpp/oneapi/dal/test/engine/dataframe_actions.hpp

```diff
@@ -94,6 +94,47 @@ class dataframe_builder_action_fill_uniform : public dataframe_builder_action {
     std::int64_t seed_ = 7777;
 };
 
+class dataframe_builder_action_fill_normal : public dataframe_builder_action {
+public:
+    explicit dataframe_builder_action_fill_normal(double a, double b, std::int64_t seed)
```

Comment:
I repeat the same question again: what is `a` and `b` for normal distribution? Use more appropriate naming. For example: `loc` and `std`.